app/routes/ics-feed[.ics].ts

```diff
@@ -6,27 +6,30 @@ import { generateIcs } from "ics-service";
 import { site } from "~/config";
 
 export const loader: LoaderFunction = async () => {
-	const events: EventAttributes[] = (await import("../data/events.json")).map(
-		(event) => {
-			const date = new Date(event.date);
-			return {
-				// TODO: add uid: event.uid,
-				title: event.topics.join(", "),
-				location: event.location,
-				url: event.link,
-				start: [
-					date.getUTCFullYear(),
-					date.getUTCMonth() + 1,
-					date.getUTCDate(),
-					date.getUTCHours(),
-					date.getUTCMinutes(),
-				],
-				startInputType: "utc",
-				endInputType: "utc",
-				duration: event.duration as DurationObject,
-			};
-		}
-	);
+	const eventData = (await import("../data/events.json")).default;
+	eventData.reverse();
+	const events: EventAttributes[] = eventData.map((event, index) => {
+		const date = new Date(event.date);
+		return {
+			// TODO: add uid: event.uid,
+			title: event.topics.join(", "),
+			location: event.location,
+			url: event.link,
+			start: [
+				date.getUTCFullYear(),
+				date.getUTCMonth() + 1,
+				date.getUTCDate(),
+				date.getUTCHours(),
+				date.getUTCMinutes(),
+			],
+			startInputType: "utc",
+			endInputType: "utc",
+			duration: event.duration as DurationObject,
+			// HACK: this relies on position in the array not changing… but
```

Comment:
This should probably work fine, but if we want to bulletproof it, what do you think about adding a unique string to each object on the events.json? For example: 'open-source' for the "contributing to open source" event, another approach might be using the date ISO string, which should be unique enough.

I'm not sure how much value it will add if we won't rely on the array index, because I assume the events are sorted according to occurrence, but worth bringing that up :)